### DISCUSSION TITLE
Combining the two themes

### DISCUSSION THREAD

**Discussion by yobson** [srid/emanote#234] [Q&A] [answered]
Hello!

Is there a way to use the book template (side bar all) with the zettelkasten style back-links?

**Comment by srid** [comment, marked as answer]
Can you elaborate? Backlinks are already available in both the themes.

**Reply by yobson** [reply]
Apologies. I meant the graph at the top of the note template. Is there a way of enabling that in the book template

**Reply by srid** [reply]
You can create a new layout in your own notebook at `/tempates/layouts/mine.tpl`, and then copy paste whatever you need from one or both the files in https://github.com/srid/emanote/tree/master/default/templates/layouts 

Then specify that template in your `index.yml`.

**Reply by srid** [reply]
Basically copy `book.tpl` and add the following somewhere inside it:

```html
<apply template="components/note-uptree" />
```

**Reply by yobson** [reply]
Perfect! Thank you so much

**Comment by srid** [comment]
@yobson This is now the default. We have one template that has both. 

https://emanote.srid.ca/guide/html-template/uptree

**Reply by yobson** [reply]
Oh this is cool! It looks great!
